src/components/reusable/widget/widget.ts

```diff
@@ -0,0 +1,91 @@
+import { LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import Styles from './widget.scss';
+
+/**
+ * Widget.
+ * @slot unnamed - Slot for widget content.
+ * @slot action - Slot for action buttons.
+ * @slot tooltip - Slot for tooltip in header.
+ * @slot draghandle- Slot for drag handle.
+ */
+@customElement('kyn-widget')
+export class Widget extends LitElement {
+  static override styles = Styles;
+
+  /** Widget title. */
+  @property({ type: String })
+  widgetTitle = '';
+
+  /** Widget sub-title. */
+  @property({ type: String })
+  subTitle = '';
+
+  /** Widget drag active state. */
+  @property({ type: Boolean })
+  dragActive = false;
```

Comment:
What else `dragActive` do apart from adding shadow ?

Reply:
That's it.